### PR TITLE
Update `voici-core`

### PR DESCRIPTION
**Pull request by jtpio**
Update to Voici 0.6.0

### CODE REVIEW

File: build-environment.yml [hunk 1]
```diff
@@ -5,5 +5,5 @@ dependencies:
   - python
   - pip
   - pip:
-    - jupyterlite-xeus>=0.1.5,<0.2
-    - voici>=0.5,<0.6
+    - jupyterlite-xeus>=0.1.8,<0.2
+    - voici-core>=0.6,<0.7
```
Comment (martinRenou) [inline build-environment.yml:9]: Actually, it could be the following?

```suggestion
    - voici=0.6,<0.7
```

Comment (jtpio) [reply build-environment.yml:9]: Yes. Wondering if leaving the two `voici-core` and `jupyterlite-xeus` dependencies would make it easier to use the Pyodide kernel instead if folks want to.

Comment (jtpio) [reply build-environment.yml:9]: So they could replace the `jupyterlite-xeus` by `jupyterlite-pyodide-kernel`.

Comment (martinRenou) [reply build-environment.yml:9]: Sounds like a good idea yes 👍🏽 let's keep it that way then

**Comment by martinRenou** [comment]
Thanks!

**Pull request merged**